packages/evergreen-component-mapper/src/text-field/text-field.tsx

```diff
@@ -1,25 +1,33 @@
 import React from 'react';
 
-import { TextInputField } from 'evergreen-ui';
+import { Autocomplete, TextInputField } from 'evergreen-ui';
 
 import useFieldApi from '@data-driven-forms/react-form-renderer/use-field-api';
 import { UseFieldApiProps } from '@data-driven-forms/react-form-renderer';
 
 export interface TextFieldProps extends UseFieldApiProps<string> {
-    name: string;
-    isRequired?: boolean;
-};
+	name: string;
+	items: string[];
```

Comment:
I think this should be optional in case anyone wants to use only the text-field.

Reply:
Agreed. Will change this.